components/board/mcp3008.go

```diff
@@ -14,7 +14,7 @@ type MCP3008AnalogReader struct {
 	Chip    string
 }
 
-// MCP3008AnalogConfig describes the configuration of a MCP3008 analog reader on a board.
+// MCP3008AnalogConfig describes the configuration of an analog reader on a board.
```

Comment:
...why change this line? The old version was more explicit/specific, though I grant it was also somewhat redundant.

Reply:
I think this was a merging issue, can change back to the old comment.